src/dialog-wrapper.component.ts

```diff
@@ -1,68 +1,80 @@
-import {
-  Component, ViewContainerRef, ViewChild, ComponentFactoryResolver, ReflectiveInjector, Type
-} from '@angular/core';
-import {DialogComponent} from "./dialog.component";
-import {DialogService} from "./dialog.service";
+import { Component, ComponentFactoryResolver, OnDestroy, ReflectiveInjector, Type, ViewChild, ViewContainerRef } from '@angular/core';
 
+import { DialogComponent } from './dialog.component';
+
+/**
+ * The modal backdrop wrapping wrapper to the modal
+ */
 @Component({
   selector: 'dialog-wrapper',
   template: `
-    <div #container class="modal fade" style="display:block !important;" role="dialog">
-        <ng-template #element></ng-template>
+    <div #wrapper class="modal fade" style="display:block !important;" role="dialog">
```

Comment:
This line could also change the class name from `modal` to `dialog`.

The `fade` class could also be removed. It's being used to apply the animation for entry-leave like `.fade.in {...}` wich could be easily done with `.dialog.in {...}`.